python/tests/onnx_quantizer_tests/layers_tests/test_e2e.py

```
@@ -128,17 +128,30 @@ def test_e2e_quantize_compile_witness_prove_verify(
+            model.base_testing(
+                CircuitExecutionConfig(
+                    run_type=RunType.GEN_WITNESS,
+                    dev_mode=False,
+                    witness_file=temp_witness_file,
+                    circuit_path=str(temp_circuit_path),
+                    input_file=temp_input_file,
+                    output_file=temp_output_file,
+                    write_json=True,
+                ),
+            )
+        except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)